src/main/java/frc/trigon/robot/components/XboxController.java

```diff
@@ -71,11 +75,13 @@ public void setDeadband(double deadband) {
     }
 
     public void rumble(double durationSeconds, double power) {
-        if (stopRumblingCommand != null)
-            stopRumblingCommand.cancel();
-        stopRumblingCommand = Commands.getDelayedCommand(durationSeconds, () -> getHID().setRumble(GenericHID.RumbleType.kBothRumble, 0));
-        stopRumblingCommand.schedule();
-        getHID().setRumble(GenericHID.RumbleType.kBothRumble, power);
+        new Trigger(() -> true).onTrue(new StartEndCommand(
+                () -> {
```

Comment:
1. Remove the trigger, use `.schedule`.
2. Make a startendcommand that starts by rumbling and ends with stopping the rumble.
3. Give it a timeout (`.withTimeout`) of the duration.